### DISCUSSION TITLE
awsebcli설치가 계속 안되서 여쭤봅니다.

### DISCUSSION THREAD

**Discussion by ho9924** [fsoftwareengineer/todo-application#25] [질문과 답변] [answered]
awsebcli설치가 계속 안되서 여쭤봅니다.
아래 사진처럼 콘솔에 찍어보니 pip, python설치 잘 되있고 환경 변수 설정도 잘 되있는 것 같습니다.
![1](https://user-images.githubusercontent.com/42732440/145752818-892e6054-a09b-48d7-b141-79bade04cdbb.png)

그런데 아래처럼 eb명령어만 입력하면 에러가 납니다.
![2](https://user-images.githubusercontent.com/42732440/145752925-ae0b6d3c-fcda-4631-ba4a-2c290e302f6f.png)

아래는 환경 변수 설정 입니다.
![53](https://user-images.githubusercontent.com/42732440/145752962-d897b4d0-05cb-4eb3-be5b-0aa914b87792.png)

저번에 알려주신 것처럼 uninstall후 다시 install해보고 환경 변수도 다시 해보고 여러 방안 검색해 보았는데 안되서 여쭤봅니다. 이것 때문에 진도가 안나가네요.. 도와주세요..

**Comment by fsoftwareengineer** [comment, marked as answer]
Python과 eb 관련 환경 변수를 삭제후 재설정 해보실수 있나요? 어떤 이유에서인지 환경변수가 제대로 불러지지 않아 리셋해야 할 수 있다고 합니다. 

다른 방법은 python 3.8버전을 사용해보는 방법이 있을 것 같습니다. 

두 방법 다 안된다면 댓글 남겨주세요!

**Reply by fsoftwareengineer** [reply]
또 사진상 script로 가는 환경 변수가 2개가 보이는데 이 설정이 맞는지 확인 해 주시기 바랍니다.

**Reply by fsoftwareengineer** [reply]
해결하셨다니 다행입니다!
